lib/const.rb

```diff
@@ -1,11 +1,16 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.0.6'
+CREW_VERSION = '1.1.0'
 
 ARCH = `uname -m`.strip
 ARCH_LIB = if ARCH == 'x86_64' then 'lib64' else 'lib' end
 
-CREW_PREFIX = '/usr/local'
+if ENV['CREW_PREFIX'].to_s == ''
+  CREW_PREFIX = '/usr/local'
+else
+  CREW_PREFIX = ENV['CREW_PREFIX']
+  @pkg.build_from_source = true
```

Comment:
This is painful in so many ways. :dizzy_face:

Reply:
How so?

Reply:
Uh, you are requiring the user to compile EVERYTHING from source?

Reply:
Well, yeah. You can't just install software compiled for `/usr/local` in `/some/random/directory`.